atmo/templates/atmo/dashboard.html

```diff
@@ -60,7 +60,7 @@
         {% for spark_job in user_spark_jobs %}
         <tr>
           <td class="hidden">{{ spark_job.id }}</td>
-          <td>{{ spark_job.identifier }}</td>
+          <td><a href="{% url "jobs-detail" id=spark_job.id %}">{{ spark_job.identifier }}</a></td>
```

Comment:
It'd be cool to define a `get_absolute_url` in the `SparkJob` and use that instead